nix/tenzir/default.nix

```diff
@@ -183,7 +182,7 @@
             "-DCMAKE_FIND_PACKAGE_PREFER_CONFIG=ON"
             "-DCAF_ROOT_DIR=${caf}"
             "-DTENZIR_EDITION_NAME=${lib.toUpper pname}"
-            "-DTENZIR_ENABLE_RELOCATABLE_INSTALLATIONS=${lib.boolToString isStatic}"
+            "-DTENZIR_ENABLE_RELOCATABLE_INSTALLATIONS=ON"
```

Comment:
Do we still have a use case for non-relocatable installations, or can we just remove the option and always enable it?

Reply:
We can remove it I think, but it's low prio.